ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -80,16 +80,16 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-serial
-- as: e2e-aws-techpreview-featuregate
-  optional: true
-  steps:
-    cluster_profile: aws
-    workflow: openshift-e2e-aws-techpreview
 - as: verify
   commands: |
     make verify
   container:
     from: src
+- as: bootstrap-e2e
+  optional: true
```

Comment:
test is failing. Looks like ci-operator-config expects values in alphabetical order. Also, I am wondering if you are fine with moving optional field into presubmits file to maintain consistency.

Reply:
> Also, I am wondering if you are fine with moving optional field into presubmits file to maintain consistency.

It was my understanding that the optional field was moved out to the ci-config to make it more explicit? There are other examples of `optional` being in the ci-config within this file (most seem to have it here)

Reply:
you are right, after looking carefully I do see it has been moved to ci-config. Looks like I missed when this change was made.